src/getModifier.ts

```diff
@@ -0,0 +1,37 @@
+import { keyToModifierBitMap } from "./keyToModifierBitMap";
+
+/**
+ * Gets the modifier bit masks based on the given option flags.
+ * This can be used to build the 'modifiers' property for CDP input dispatch events.
+ * @param {Object} options - The options object
+ * @param {boolean} [options.shiftKey] - Whether the Shift key is pressed
+ * @param {boolean} [options.altKey] - Whether the Alt key is pressed
+ * @param {boolean} [options.ctrlKey] - Whether the Control key is pressed
+ * @param {boolean} [options.metaKey] - Whether the Meta key is pressed 
+ * @returns {number} The modifier bit mask flags
+ */
+export function getModifier(options: {
```

Comment:
I think this doesn't work as expect. You should be able to stack them, I suppose it should work like a bit mask so it is possible to pass the shift + alt + control keybinding. 

Would be good to add this test first and then try to fix it

Reply:
I didn't try to cover modifiers combination. Only ability to work with shift or alt or ctrl or meta separately

Reply:
I didn't go that deep, because I do not fully understand how bit map works. If you could help me with that, I would love to update the code.

Reply:
@dmtrKovalenko your requested changes fulfilled. All checks are passed. Please, review.